### ISSUE TITLE
Not working in 6.3

### ISSUE THREAD

**Issue by gguga12** [goaaats/Dalamud.LoadingImage#5]
Hey goat i love the plugin, and i can't wait for it to be updated!

**Comment by Daccura** [comment]
Would be appreciated if this was updated for 6.3

**Comment by Carighan** [comment]
Agreed. Really missing this. Is there anything we can do to help, @goaaats ?
